homeassistant/components/nmap_tracker/config_flow.py

```diff
@@ -92,23 +103,35 @@ def normalize_input(user_input):
     return errors
 
 
-async def _async_build_schema_with_user_input(hass, user_input):
+async def _async_build_schema_with_user_input(hass, user_input, include_options):
     hosts = user_input.get(CONF_HOSTS, await hass.async_add_executor_job(get_network))
     exclude = user_input.get(
         CONF_EXCLUDE, await hass.async_add_executor_job(get_local_ip)
     )
-    return vol.Schema(
-        {
-            vol.Required(CONF_HOSTS, default=hosts): str,
-            vol.Required(
-                CONF_HOME_INTERVAL, default=user_input.get(CONF_HOME_INTERVAL, 0)
-            ): int,
-            vol.Optional(CONF_EXCLUDE, default=exclude): str,
-            vol.Optional(
-                CONF_OPTIONS, default=user_input.get(CONF_OPTIONS, DEFAULT_OPTIONS)
-            ): str,
-        }
-    )
+    schema = {
+        vol.Required(CONF_HOSTS, default=hosts): str,
+        vol.Required(
+            CONF_HOME_INTERVAL, default=user_input.get(CONF_HOME_INTERVAL, 0)
+        ): int,
+        vol.Optional(CONF_EXCLUDE, default=exclude): str,
+        vol.Optional(
+            CONF_OPTIONS, default=user_input.get(CONF_OPTIONS, DEFAULT_OPTIONS)
+        ): str,
+    }
+    if include_options:
+        schema.update(
+            {
+                vol.Optional(
+                    CONF_TRACK_NEW,
```

Comment:
Why did we add this option? We already have a system level option to disable adding new entities for an integration. We're generally declining this kind of option for device trackers.

Reply:
It was an intermediate solution during beta.
However, this PR has been reverted (the full nmap tracker modernization has been pulled).